azure-pipelines.bundlesize.yml

```diff
@@ -24,25 +22,25 @@ jobs:
           filePath: yarn-ci.sh
         displayName: yarn
 
-      - script: |
-          yarn nx affected -t bundle-size --nxBail $(sinceArg)
-        displayName: build packages & create reports
-        condition: eq(variables.isPR, true)
+      # - script: |
```

Comment:
commenting out for now. 

I could have moved the needed logic to separate ADO pipeline and go through all the hassle of registration steps while enabling on ADO, which I decided not to do.